database/migrations/2025_04_01_214953_add_customization_columns.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('dashboard_layout')->default('grid');
+            $table->integer('console_font_size')->default(14);
+            $table->string('console_font')->default('monospace');
+            $table->integer('console_rows')->default(30);
+        });
```

Comment:
Let's use a panel_settings json column instead!

Reply:
Maybe, JSON sometimes is mean to me